libs/domain/search/src/services/adapter/default-suggestion.adapter.ts

```diff
@@ -1,14 +1,14 @@
 import { HttpService, JsonAPITransformerService } from '@spryker-oryx/core';
 import { inject } from '@spryker-oryx/di';
 import { ApiProductModel } from '@spryker-oryx/product';
-import { Observable } from 'rxjs';
+import { Observable, of } from 'rxjs';
 import {
   ApiSuggestionModel,
-  Suggestion,
   SuggestionQualifier,
+  SuggestionResponse,
 } from '../../models';
 import { SuggestionNormalizer } from './normalizers';
-import { SuggestionAdapter } from './suggestion.adapter';
+import { SuggestionAdapter, SuggestionField } from './suggestion.adapter';
 
 export class DefaultSuggestionAdapter implements SuggestionAdapter {
```

Comment:
Should it be named default if it's a posible one of many?

Reply:
But it's default spryker adapter. Not sure if I can use Spryker prefix in it

Reply:
`Default` prefix is indeed the convention we took for all Spryker integrations. We have considered `Glue` prefix, but i'm glad we didn't as Glue is more a framework then a specific API.

Reply:
👍